sway/config/output.c

```diff
@@ -125,6 +126,63 @@ void merge_output_config(struct output_config *dst, struct output_config *src) {
 	}
 }
 
+struct output_config *duplicate_output_config(const struct output_config *src) {
```

Comment:
Can we simplify this to the following?

- Create a new empty `dst` config
- Merge `src` into `dst`